### ISSUE TITLE
Change UI to use Blazor Server Or Wasm

### ISSUE THREAD

**Issue by masoodafar-web** [skoruba/IdentityServer4.Admin#1032]
Hello, I hope you are well.
I wanted to know if it is possible to use Blazor Server or Blazor Wasm for the admin panel?
And if we cannot use these, what is the reason?
And if we can use it, do you suggest it?
I want to use it (blazor) in an IDP, is this safe?

**Comment by skoruba** [comment]
Hi, what do you mean Blazor for admin? Rewrite mvc to blazor or use blazor app with IdP?

**Comment by masoodafar-web** [comment]
> Hi, what do you mean Blazor for admin? Rewrite mvc to blazor or use blazor app with IdP?
Hi,Yes, that's what I mean.

**Comment by vpetkovic** [comment]
@masoodafar-web he asked you to clarify which one of the two you are referring to

1. Rewrite mvc to blazor ?
2. use blazor app with IdP?

I think you are referring to the first option. One approach would be to use Management Admin API and rewrite MVC pages in Blazor. 

When it comes to Identity portion with Blazor Server, from what I understand, you still need to resort to MVC backend since you cannot post forms to Blazor Server hence and Microsoft is not planning to port Identity to Blazor.

**Comment by masoodafar-web** [comment]
Yes, I mean the first option, that is (Rewrite MVC to Blazor), is it possible to do this? If it is not possible to migrate the front from MVC to Blazor, what is the reason? Are there security reasons? For example, maybe you can use Blazor Server for this issue.

**Comment by skoruba** [comment]
What is the benefit of rewrite to Blazor instead of keeping it in MVC?

**Comment by masoodafar-web** [comment]
> What is the benefit of rewriting to Blazor instead of keeping it in MVC?

Using existing and more optimized components in Blazor, as well as working and developing (code style) more easily to customize business related codes.
